domain/user/state/state.go

```diff
@@ -256,26 +258,26 @@ WHERE user.uuid = $M.uuid AND removed = false
 		}
 
 		var result User
-		err = tx.Query(ctx, selectGetUserWithAuthStmt, sqlair.M{"uuid": uuid.String()}).Get(&result)
+		err = tx.Query(ctx, selectGetUserWithAuthStmt, sqlair.M{"name": name}).Get(&result)
 		if err != nil && errors.Is(err, sql.ErrNoRows) {
-			return errors.Annotatef(usererrors.NotFound, "%q", uuid)
+			return errors.Annotatef(usererrors.NotFound, "%q", name)
```

Comment:
Check the upstream for this. If it's user input, we might want to return `Unauthorized` unless the facade converts it.

Otherwise we're giving an opportunist some feedback.

Reply:
Not sure that I understand why...

As I understand for now we have the following logic:
- find the user in DB, if not – NonFound Error
- compare the hash from salt and in DB, if not equal – Unauthorized Error

Do we wish to change it?

Reply:
Changed. Now this function returns only `Unauthorized` error in any case.